cluster-autoscaler/provisioningrequest/provreqwrapper/wrapper.go

```diff
@@ -81,3 +81,15 @@ func errMissingPodTemplates(podSets []v1.PodSet, podTemplates []*apiv1.PodTempla
 	}
 	return fmt.Errorf("missing pod templates, %d pod templates were referenced, %d templates were missing: %s", len(podSets), len(missingTemplates), strings.Join(missingTemplates, ","))
 }
+
+// Parameters makes a deep copy of embedded ProvReq and sets its Parameters
+func (pr *ProvisioningRequest) Parameters(params map[string]v1.Parameter) *ProvisioningRequest {
+	prCopy := pr.DeepCopy()
```

Comment:
It's a test utility function, so let's move it to the testutils file.

Reply:
Done